integration_test/third_party_apps_data/applications/hbase/centos_rhel/install

```diff
@@ -0,0 +1,36 @@
+set -e
+
+sudo yum -y install \
+    java-1.8.0-openjdk java-1.8.0-openjdk-devel wget
+
+sudo mkdir /opt/hbase
+wget http://apache.mirror.gtcomm.net/hbase/stable/hbase-2.4.9-bin.tar.gz
```

Comment:
Chances are a bit higher that users will use the default package manager (e.g. yum, apt-get, zyypper) to install a package instead of a pinned version of the application.

Can we adjust it to do something like `yum install hadoop` instead? Similar requests for the other `install` scripts.

Reply:
There doesn't seem to be one for hbase either.

Reply:
Got it. Guess we have to maintain this piece of code then. NVM